src/components/Tabs/Tab/Tab.tsx

```diff
@@ -9,6 +9,8 @@ export interface TabItem {
   id?: string;
   /** Title of the tab */
   title?: string;
+  /** URL if tab is link */
+  href?: string;
 }
```

Comment:
I just noticed that we probably need to add a `target` prop as well, so that consumers can pass `_blank` for instance.